File: edudoc/src/Model/DTOs/ClaimAuditResponseDto.cs
using System.Data.SqlClient;
using System.Data.SqlClient;
using System;
using System.Collections.Generic;

namespace Model.DTOs
{
    public class ClaimAuditResponseDto
    {
        public string StudentName { get; set; }
        public DateTime DateSigned { get; set; }
        public int EncounterId { get; set; }
        public string EncounterNumber { get; set; }
        public string SchoolDistrict { get; set; }
        public string CurrentStatus { get; set; }
        public int CurrentStatusId { get; set; }
        public string ProviderName { get; set; }
        public string ServiceArea { get; set; }
        public int ServiceAreaId { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public DateTime EncounterDate { get; set; }
        public int NumStudentsInEncounter { get; set; }
        public IEnumerable<CptCode> CptCodes { get; set; }
        public IEnumerable<Goal> Goals { get; set; }
        public string TreatmentNotes { get; set; }
        public string ReasonForAudit { get; set; }
        public string MedicaidNo { get; set; }
        public StudentParentalConsent CurrentParentalConsent { get; set; }
        public bool NeedsReferral { get; set; }
        public ICollection<SupervisorProviderStudentReferalSignOff> ReferralSignOffs { get; set; }
        public IEnumerable<EncounterStudentStatusesLogDto> EncounterStudentStatuses { get; set; }

    }
}

File: edudoc/src/app/model/interfaces/custom/claim-audit-response.dto.ts
import { ICptCode } from '../cpt-code';
import { IGoal } from '../goal';
import { IEncounterStudentStatusesLogDto } from './encounter-student-statuses-log.dto';

export interface IClaimAuditResponseDto {
    DateSigned: Date;
    EncounterId: number;
    StudentName: string;
    SchoolDistrict: string;
    CurrentStatus: string;
    CurrentStatusId: number;
    ProviderName: string;
    EncounterNumber: string;
    ServiceArea: string;
    EncounterDate: Date;
    StartTime: string;
    StartDateTime: Date;
    EndTime: string;
    EndDateTime: Date;
    MedicaidNo: string;
    NumStudentsInEncounter: number;
    CptCodes: ICptCode[];
    Goals: IGoal[];
    ReasonForAudit: string;
    TreatmentNotes: string;
    EncounterStudentStatuses: IEncounterStudentStatusesLogDto[];
}
